### PR TITLE
ddl: add column info and index info in log

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@coocood @winkyao PTAL

File: ddl/column.go [hunk 1]
```diff
@@ -127,6 +127,7 @@ func (d *ddl) onAddColumn(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 			job.State = model.JobStateCancelled
 			return ver, errors.Trace(err)
 		}
+		log.Infof("[ddl] add column, run DDL job %s, column info %#v, offset %d", job, columnInfo, offset)
```
Comment (winkyao) [inline ddl/column.go]: log it after finished add index?

File: ddl/index.go [hunk 2]
```diff
@@ -231,6 +231,7 @@ func (d *ddl) onCreateIndex(t *meta.Meta, job *model.Job) (ver int64, err error)
 		indexInfo.Unique = unique
 		indexInfo.ID = allocateIndexID(tblInfo)
 		tblInfo.Indices = append(tblInfo.Indices, indexInfo)
+		log.Infof("[ddl] add index, run DDL job %s, index info %#v", job, indexInfo)
```
Comment (winkyao) [inline ddl/index.go]: ditto

File: ddl/column.go (hunk 1 above)
Comment (coocood) [reply ddl/column.go]: Or add log at start and finished?

Comment (zimulala) [reply ddl/column.go]: @winkyao 
Here is adding the column.
Why log it after finishing the operation?
@coocood 
We haven't this information in the start or finish DDL. Here is start `add column`.

**Comment by zimulala** [comment]
PTAL @winkyao @coocood

**Pull request merged**